lib/webauthn/attestation_statement/packed.rb

```diff
@@ -15,12 +15,16 @@ def valid?(authenticator_data, client_data_hash)
         valid_format? &&
           valid_certificate_chain?(authenticator_data.credential) &&
           meet_certificate_requirement? &&
+          matching_aaguid?(authenticator_data.aaguid) &&
           valid_signature?(authenticator_data, client_data_hash) &&
           attestation_type_and_trust_path
       end
 
       private
 
+      AAGUID_LENGTH = WebAuthn::AuthenticatorData::AttestedCredentialData::AAGUID_LENGTH
+      private_constant :AAGUID_LENGTH
```

Comment:
I think I prefer not to copy constants from one module to the other.

I have no problem with referencing directly despite having a long name.

If you prefer to use a shortcut for improving readability of the code, we could either put it in a local variable or define a private method in this class that returns the "long" constant.

Reply:
I think I'll change to a local variable in that case.